src/utils/common.js

```diff
@@ -591,18 +591,26 @@ export const getDeviceWidth = () => {
   return Dimensions.get('window').width;
 };
 
-export const formatTransactionFee = (feeInWei: string | number, gasToken: ?GasToken): string => {
-  if (!feeInWei) return '';
+export const getFormattedTransactionFeeValue = (feeInWei: string | number | BigNumber, gasToken: ?GasToken): string => {
+  // fixes exponential values with BigNumber.toPrecision()
+  const parsedFeeInWei = typeof feeInWei === 'object' // BigNumber
+    ? feeInWei.toPrecision()
```

Comment:
How about putting a note (todo), that we should replace this to `feeInWei.toFixed()` after we upgrade to the newest Bignumber version? As this will be a breaking change

Reply:
👍